manifests/charts/gateway/values.yaml

```diff
@@ -73,6 +73,7 @@ autoscaling:
   minReplicas: 1
   maxReplicas: 5
   targetCPUUtilizationPercentage: 80
+  targetMemoryUtilizationPercentage: 80
```

Comment:
I think you missed turning this one off by default?

Reply:
I think I missed one part... Thank you. I'll fix it :)